cirq-core/cirq/ops/classically_controlled_operation.py

```diff
@@ -45,6 +45,22 @@ class ClassicallyControlledOperation(raw_types.Operation):
 
     This object is typically created via
      `operation.with_classical_controls(*conditions)`.
+
+    Examples:
+        q0, q1, q2 = cirq.LineQubit.range(3)
```

Comment:
Can you check some of the other code in the repository to be consistent with the style of other code examples?  

I don't think this will render quite right in the doc site.  I think, to look like code, we need to either add triple backquotes or a ">>> " at the beginning for a code example.  I am not totally sure though.

Reply:
My bad, should have aligned with the REPL/CLI-style format. In this iteration, I'm using the `circuit.py` file as the example: https://github.com/quantumlib/Cirq/blob/0361a81c89ff47656451ada08d99412d5c128e16/cirq-core/cirq/circuits/circuit.py#L1431